src/main/java/mate/academy/spring/boot/model/Book.java

```diff
@@ -5,17 +5,21 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Getter;
 import lombok.Setter;
+import lombok.ToString;
 import org.hibernate.annotations.SQLDelete;
 import org.hibernate.annotations.SQLRestriction;
 
 @Getter
 @Setter
+@ToString
```

Comment:
it's ok for now, but be careful with related entities, can cause Lazy Init exception